pinot-core/src/test/java/org/apache/pinot/queries/TextSearchQueriesTest.java

```diff
@@ -1864,6 +1878,26 @@ public void testInterSegment() {
     query = "SELECT count(*) FROM MyTable WHERE TEXT_MATCH(SKILLS_TEXT_COL_DICT, 'a and or in the are')";
     testInterSegmentAggregationQueryHelper(query, 0);
 
+    // query with excluded stop-words. they should be indexed
+    query = "SELECT count(*) FROM MyTable WHERE TEXT_MATCH(SKILLS_TEXT_COL, '\"IT support\" or \"IT manager\"')";
+    testInterSegmentAggregationQueryHelper(query, 8);
+
+    // query with excluded stop-words. they should be indexed
+    query = "SELECT count(*) FROM MyTable WHERE TEXT_MATCH(SKILLS_TEXT_COL, '\"IT\"')";
+    testInterSegmentAggregationQueryHelper(query, 16);
+
+    // query without stop-words
+    query = "SELECT count(*) FROM MyTable WHERE TEXT_MATCH(SKILLS_TEXT_COL, '\"support\" or \"manager\"')";
+    testInterSegmentAggregationQueryHelper(query, 12);
```

Comment:
Can we add a test where for another text index column, we don't set the excludeStopWords IT in the FieldConfig and thus the result of search `'\"IT support\" or \"IT manager\"'` is same as `'\"support\" or \"manager\"'`  which is 12

Reply:
Good point. Added